Automattic/wp-calypso
# Purchases: Add option to remove expiring purchases

Fixes #1402 by adding an option to `ManagePurchase` that opens a dialog to remove an expiring purchase.

![screen shot 2016-01-19 at 4 59 25 pm](https://cloud.githubusercontent.com/assets/1130674/12436941/559b0234-bece-11e5-87cf-1ed0aef99559.png)

**Testing**
- Point your hosts file at your sandbox.
- Apply this patch: D906-code
- Visit `/purchases/:site/:purchase_id` for a site/purchase that has auto-renew disabled and is outside the refund period.
- You should see an option to 'Remove [product]'. Clicking it will open the dialog above.
- Click 'Remove' and assert that the buttons are disabled, and that you are eventually redirected to `/purchases` with a notice indicating that the purchase was removed.
- Assert that the purchase that was removed no longer appears in the list below.
